app/containers/WalletInfo/TokenBalances.jsx

```diff
@@ -39,7 +40,7 @@ const TokenBalances = ({ tokenBalances, showModal }: Props) => (
               <Tooltip
                 title={formattedBalance}
                 disabled={isZero(balance)}>
-                {formattedBalanceDisplay}
+                {formattedBalanceDisplay} {deprecated && <span style={{ color: 'red' }}>(deprecated)</span>}
```

Comment:
Would prefer that we use a className (via `TokenBalances.scss`) instead of inline styles here.